aten/src/ATen/native/native_functions.yaml

```diff
@@ -3621,18 +3621,30 @@
   python_module: nn
 
 # Return: (Tensor output, Tensor indices)
-- func: adaptive_max_pool2d(Tensor self, int[2] output_size, *, Tensor(a!) output, Tensor(b!) indices) -> (Tensor(a!), Tensor(b!))
+- func: adaptive_max_pool2d(Tensor self, int[2] output_size, *, Tensor(a!) out, Tensor(b!) indices) -> (Tensor(a!), Tensor(b!))
```

Comment:
Huh, what happened here?

Reply:
That was me being in literal mode while trying to debug the `test_jit` issue. :) I was grasping at straws, then I saw that the README  recommends `out` as the convention, so I checked if the JIT by any chance relies on the argument name convention.

Reply:
@skrah Nice catch. Could you add this to the wiki? https://github.com/pytorch/pytorch/wiki/TH-to-ATen-porting-guide You should have edit rights.

Reply:
@ezyang  Yes, I added a small entry to the wiki.